README.md

```diff
@@ -11,15 +10,63 @@ logic.
     msgid "user"
     msgstr "użytkownika"
 
+With ``AttributiveTranslation`` class you can access ``accusative`` context
+translation through attribute of no-context translation:
```

Comment:
I don't understand the purpose of "of no-context translation" here.
Isn't "no-context translation" supposed to mean ()->"użytkownik"?

Reply:
By no-context I mean result of gettext call. In opposite to context translation (which by default is result of pgettext function call).